modules/Internals/Tools/java-dump.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+function decompile {
```

Comment:
Hi,

Thank you for the patch.

Why not just adding '-s -c -private -verbose' to javap call in the `showFile` function?

Reply:
I used the `-c` flag at first, but we're running a diff against the full version of OpenJDK's `rt.jar`, so this utility provides a more condensed summary. I'll re-raise a PR and include two alternative options for this; summarised and full method bodies. The former, which is what I'm using here, essentially strips any numeric references from the bytecode and runs SHA256 on the set of instructions that make up each method body.

Reply:
FYI about OpenJDK testing:

This page may be of interest to you: https://abi-laboratory.pro/java/tracker/timeline/openjdk/

Thank you.

Reply:
Yep, I saw that one as well. But thanks for pointing me to it. :+1: